docs/core/tutorials/using-on-macos.md

```diff
@@ -54,10 +52,10 @@ This command creates a *golden.sln* file for the entire solution.
 
 Your next task is to create the library. In the terminal window
 (either the embedded terminal in VS code, or another terminal),
-cd to *golden/* and type the command:
+change directory (`cd`) to *golden* and type the command:
```

Comment:
maybe to make this a bit smoother:
change the directory to *golden* with the following command:
`cd golden`
Then type the command: